video/out/gpu/video.c

```diff
@@ -1291,8 +1299,22 @@ static void copy_image(struct gl_video *p, int *offset, struct image img)
         img.multiplier *= 1.0 / (tex_max - 1);
     }
 
-    GLSLF("color.%s = %f * vec4(texture(texture%d, texcoord%d)).%s;\n",
-          dst, img.multiplier, id, id, src);
+    if (img.interlaced) {
```

Comment:
As @bjin pointed out, `copy_image` is not the best place to have this functionality. It probably belongs in the beginning of `pass_read_video`. I'm not sure what the best place is to do deinterlacing, but we probably want to do it as one of the very first operations (before debanding, crucially).